Pheno/scripts/anova_postHOC.R

```diff
@@ -0,0 +1,81 @@
+library(tidyverse)
+library(DescTools) 
+
+# Running ANOVA test on KhoeSan pigmentation averages
+
+# set work directory
+setwd("/Users/dana.alhindi/DDRIG/Pheno/data_private/")
+
+# Load data
+cdb <- read.csv(file = "MasterPheno_CDB_DRA.csv")
+karoo <- read.csv(file = "MasterPhenotpe_Karoo_DRA.csv")
+nama <- read.csv(file = "MasterPhenotype_Nama.csv")
+kho <- read.csv(file = "MasterPhenotype_KhmAll.csv")
+data <- read.csv(file = "pigmentation_data.csv")
+data_omitKaroo <- read.csv(file = "pigmentation_data_minusKaroo.csv")
+
+
+# Removing empty rows
+cdb <- cdb[-which(is.na(as.numeric(cdb$M_Arm))),]
+nama <- nama[-which(is.na(as.numeric(nama$M_Arm))),]
+kho <- kho[-which(is.na(as.numeric(kho$M_Arm))),]
+data <- data[-which(is.na(as.numeric(data$M_Arm))),]
+data_omitKaroo <- data_omitKaroo[-which(is.na(as.numeric(data_omitKaroo)))]
+
+# Making sure columns are numeric
+cdb$M_Arm <- as.numeric(cdb$M_Arm)
+karoo$M_Arm <- as.numeric(karoo$M_Arm)
+nama$M_Arm <- as.numeric(nama$M_Arm)
+kho$M_Arm <- as.numeric(kho$M_Arm)
+data$M_Arm <- as.numeric(data$M_Arm)
+data_omitKaroo$M_Arm <- as.numeric(data_omitKaroo$M_Arm)
```

Comment:
hmmm this looks very copy / paste and like a lot of manual labor. Is there a way you can simply look over everything you need to do and do the same thing to each thing one by one?

maybe through some kind of looping function, like apply or lapply, or something else?

Reply:
maybe, in the future...